src/lender/assessor.sol

```diff
@@ -81,17 +82,17 @@ contract Assessor is Auth, FixedPoint, Interest  {
         else {revert("unknown-variable");}
     }
 
-    function updateSeniorAsset(uint seniorRatio_, uint seniorSupply, uint seniorRedeem) external auth {
-        dripSeniorDebt();
-
-        uint seniorAsset = safeSub(safeAdd(safeAdd(seniorDebt_, seniorBalance_), seniorSupply), seniorRedeem);
-
+    function _rebalance(uint seniorAsset_, uint seniorRatio_) internal {
         // re-balancing according to new ratio
         // we use the approximated NAV here because during the submission period
         // new loans might have been repaid in the meanwhile which are not considered in the epochNAV
         seniorDebt_ = rmul(navFeed.approximatedNAV(), seniorRatio_);
-        seniorBalance_ = safeSub(seniorAsset, seniorDebt_);
+        seniorBalance_ = safeSub(seniorAsset_, seniorDebt_);
+    }
 
+    function updateSeniorAsset(uint seniorRatio_) external auth {
+        dripSeniorDebt();
+        _rebalance(safeAdd(seniorDebt_, seniorBalance_), seniorRatio_);
```

Comment:
Would be helpful to rename the state vars seniorDebt_ & seniorBalance_ -> can otherwise be confused with function variables.

Reply:
we have a function called `seniorDebt()` thats the reason why i have this convention